tutorials/monitoring/setup/index.md

```diff
@@ -0,0 +1,208 @@
+---
+title: "Monitoring NServiceBus solutions: Getting started"
+reviewed: 2017-11-07
+summary: Setting up the Particular Service Platform to monitor an NServiceBus system.
+component: Core
+versions: '[5,)'
+---
+
+This tutorial will guide you through the steps required to set up your NServiceBus solution for monitoring with the Particular Service Platform.
+
+
+## Component overview
+
+```mermaid
+graph TD
+
+subgraph Endpoint
+  nsb(NServiceBus) --> metrics
+  metrics(NServiceBus.Metrics) --> plugin
+
+  plugin(NServiceBus.Metrics.ServiceControl)
+end
+
+plugin --> q(Monitoring Queue)
+
+nsb --> auditQ(Audit Queue)
+nsb --> errorQ(Error Queue)
+
+subgraph ServiceControl
+  scinstance
+  scmonitoring
+end
+
+auditQ --> scinstance(ServiceControl Instance)
+errorQ --> scinstance
+
+q --> scmonitoring[Monitoring Instance]
+
+scmonitoring --> pulse[ServicePulse]
+scinstance --> pulse
+
+```
+
+![the above renders like this](component-overview.png)
+
+Every system being monitored will push data to three main queues:
+
+- Monitoring: Each endpoint will gather and send metric data to this queue. This data holds information about the health and performance of an endpoint.
+- Audit: When an endpoint successfully processes a message, a copy of the message will be forwarded to this queue. The audited message will have additional headers which contain data about the processing of the message.
+- Error: When an endpoint fails to process a message (after exhausting automated retry attempts), a copy of the message is fowarded to this queue. The failed message will have additional headers which contain data about the failed attempt to process the message.
+
+The core NServiceBus package can be configured to send messages to the Audit and Error queues. In order to send data to the Monitoring queue, two additional packages are required:
+
+- NServiceBus.Metrics: hooks into the NServiceBus pipeline to gather and expose metrics.
+- NServiceBus.Metrics.ServiceControl: buffers the data generated by NServiceBus.Metrics and periodically sends it to the monitoring queue. This package also instruments outgoing messages with additional headers that are used to estimate queue length.
+
+ServiceControl is a suite of back-end tools that collect useful information about your running system. A monitoring system back-end consists of two different components:
+
+- ServiceControl instance: collects messages from the audit and error queues. Error messages are grouped together and can be retried in bulk.
```

Comment:
should we link "ServiceControl instance' to https://docs.particular.net/servicecontrol/servicecontrol-instances/ ?

Reply:
> Error messages are grouped together and can be retried in bulk.

I'd skip that part here, as that is SP functionality, may be confusing. Or modify, e.g. "The collected error messages are grouped and then can be retried from the SP dashboard"